docs/source/cuda/cuda_compilation.rst

```diff
@@ -40,22 +52,28 @@ ABI device functions requires three issues to be addressed:
   of the return value of the function. For use of Numba-compiled functions
   outside of Numba, this can generally be ignored.
 
-A simple way to address all these issues is to compile device functions with the
-C ABI instead. This results in the following:
+A simple way to address all these issues is to compile device functions with
+the C ABI instead. This results in the following:
 
-- The name of the compiled device function in PTX can be controlled. By default
-  it will match the name of the function in Python, so it is easy to determine.
-  This is the function's ``__name__``, rather than ``__qualname__``, because
-  ``__qualname__`` encodes additional scoping information that would make the
-  function name hard to predict, and in a lot of cases, an illegal identifier
-  in C.
+- The name of the compiled device function in the generated code can be
```

Comment:
```suggestion
- The name of the compiled device function in the generated output can be
```

Reply:
I think changing this to "The name of the device function in the compiled code" is clearer still (IMHO :slightly_smiling_face:) and also consistent with changing to "compiled code" above, so I'm going to go with that.

Reply:
Agree, thanks!